draft-ietf-mops-streaming-opcons.md

```diff
@@ -341,6 +371,60 @@ We note that other operators saw similar spikes during this time period. Craig L
 - A 30% increase in upstream traffic over their pre-pandemic levels, and
 - A steady increase in the overall volume of DDoS traffic, with amounts exceeding the pre-pandemic levels by 40%. (He attributed this increase to the significant rise in gaming-related DDoS attacks ({{LabovitzDDoS}}), as gaming usage also increased.)
 
+# Latency Considerations
+
+Streaming media latency refers to the "glass-to-glass" time duration, which is the delay between the real-life occurence of a recorded event and the media being appropriately displayed on an end user's device.  Note that this is different from the network latency (defined as the time for a packet to cross a network from end to end) because it includes video encoding time, and for most cases also ingest to an intermediate service such as a CDN or other video distribution service, rather than a direct connection to an end user.
```

Comment:
The SVA work that was discussed at IETF 108 is now published -- https://streamingvideoalliance.docsend.com/view/mqn3g2hvdatwp63p .  It might be worth referencing here just as a "further reading".   It is far, far, far too detailed for the discussion

Reply:
It's also been published now by the SVA since the IETF108 session. Here's the link:   https://www.streamingvideoalliance.org/document/best-practices-for-end-to-end-workflow-monitoring/

Perhaps reference both?

Reply:
I think that's what I was trying to reference -- better link :^)

Reply:
sorry, I was reading the comments via email and my mail system rewrites URLs - so we can be safe as Dora the Explorer would say except without the irony.   In any case I didn't see that ldaigle's link was also the SVA.   The one I provided is to the wrapper page for the doc which allows download plus gives information about the doc and the work around it. So as ldaigle says it's the slightly better one to use.     

And thanks to email Dora for keeping URLs inside emails "safe" and thoroughly hard to read.

Reply:
Great minds...
https://github.com/ietf-wg-mops/draft-ietf-mops-streaming-opcons/pull/27#discussion_r513605564
> If we do want to mention workflow stages here, the recent SVA document, "Best Practices for End-to-End Workflow Monitoring" may be a useful reference.

...was also a link to that same page. 😄

Reply:
SVA references will be handled/incorporated separately.